docs/uploader/aws-s3.mdx

```diff
@@ -434,7 +434,7 @@ uppy.use(AwsS3, {
 ```
 
 See the
-[aws-presigned-url example in the uppy repository](https://github.com/transloadit/uppy/tree/main/examples/aws-presigned-url)
+[aws-php example in the uppy repository](https://github.com/transloadit/uppy/tree/main/examples/aws-php)
```

Comment:
The Node.js example is probably more up-to-date
```suggestion
[aws-nodejs example in the uppy repository](https://github.com/transloadit/uppy/tree/HEAD/examples/aws-nodejs)
```
Although we could certainly link to both.